plugins/chat/spec/models/chat/direct_message_channel_spec.rb

```diff
@@ -53,8 +53,8 @@
   end
 
   describe "#threading_enabled" do
-    it "defaults to true" do
-      expect(channel.threading_enabled).to be(true)
+    it "defaults to false" do
+      expect(channel.threading_enabled).to be(false)
```

Comment:
This could be written as `expect(channel).not_to be_threading_enabled`, not sure which one reads best.